lib/Horde/Db/Adapter/Postgresql/Schema.php

```diff
@@ -1239,11 +1239,11 @@ public function pkAndSequenceFor($table)
 
             // Return null if no results or multiple rows
             // (multiple rows means composite PK with multiple columns where autoincrement is not supported )
-            if (!$results || count($results) > 1) {
+            if (!$result || count($result) > 1) {
                 return [null, null];
```

Comment:
We should probably turn this into an object rather than an array of results in a later breaking change.